.gitignore

```diff
@@ -6,6 +6,8 @@
 /cnf/GAP-LDFLAGS
 /cnf/GAP-LIBS
 /cnf/GAP-OBJS
+/cnf/GAP-KERNEL_MAJOR_VERSION
+/cnf/GAP-KERNEL_MINOR_VERSION
```

Comment:
What is the purpose of these two files?

Reply:
I was following the pattern of existing lines in sysinfo.gap -- we make a file which contains the variable, use some little trick to make sure we only touch the file if it changes, and then make sysinfo.gap depend on those new files.

Reply:
That's not what those are for: `sysinfo.gap` already depends on `config.status`, so if the kernel version is changed in `configure.ac`, we'll automatically regenerate `configure`, then re-run it, then `config.status` is called to regenerate `sysinfo.gap`

Those extra rules for CFLAGS, OBJS etc. exist because they can be changed by a change to the `Makefile` alone: e.g. adding or remove a source file can change `OBJS`, and the user might do `make CFLAGS=-O3` to recompile GAP with different compiler flags.

But we certainly don't want to allow users to do a `make KERNEL_MAJOR_VERSION=42`.

Hence, unless I am missing something, I think these files and all the related code should be removed.

Reply:
Thanks for explaining, will fix